apache-arrow/README.md

```diff
@@ -1,3 +1,13 @@
+### Why We Mirror Apache Arrow
+
+Groonga depends on Apache Arrow. However, the official Apache Arrow repository has been unstable at times, causing issues for Groonga users. To resolve this situation, we have been providing a mirrored version for Groonga users.
+
+#### Note
+
+We would like to stop mirroring the Apache Arrow repository.
+Therefore, we intend to stop mirroring once the Apache Arrow repository is stable.
+
+
```

Comment:
```suggestion
```

Reply:
[6c11b3f](https://github.com/groonga/packages.groonga.org/pull/40/commits/6c11b3ff4dd04a8ae959ace89043aff422bb9768) Thank you for catching it 🙏🏾